assets/wizards/popups/views/segments/SegmentsList.js

```diff
@@ -296,6 +308,26 @@ const SegmentsList = ( { wizardApiFetch, segments, setSegments, isLoading } ) =>
 				setSegments( segments );
 			} );
 	};
+	const generateDefaultSegments = () => {
+		setInFlight( true );
+		setError( null );
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-popups-wizard/segmentation-defaults',
+			method: 'POST',
+			quiet: true,
+		} )
+			.then( _segments => {
+				setSegments( _segments );
+			} )
```

Comment:
Simpler version, since `setSegments` is unary: 
```
.then( setSegments )
```

Reply:
6515f25 moves `setDefaultsCreated( true );` to this promise handler so that the button label doesn't update if the initial creation fails.